doc/version_history.rst

```diff
@@ -11,6 +11,8 @@ v0.25.0
 
 * Add new script ``latiss_acquire.py`` for AuxTel.
   This script is used to slew to a target and center it at a specific position.
+* In ``auxtel/correct_pointing`` and ``auxtel/latiss_base_align``, add config to search local catalog and set
+  to HD_cwfs_stars by default. 
```

Comment:
This should be in the line above. We use semantic line breaking in docs so each phrase should be in a single line.